lib/puppetserver/ca/puppet_config.rb

```diff
@@ -179,6 +187,14 @@ def run(command)
         %x( #{command} )
       end
 
+      def munge_ttl_setting(ca_ttl_setting)
```

Comment:
Can we move the constants `TTLSetting::UNITMAP` and `TTLSetting::FORMAT` into this class and then make this method have the same body as `TTLSetting#munge` and then delete the `TTLSetting` class?